platform/docs/docs/platform/services/ui/ui-customization-service.md

```diff
@@ -0,0 +1,256 @@
+---
+sidebar_position: 3
+sidebar_label: UI Modal Service
+---
+# UI Customization Service
+
+There are a lot of places where users may want to configure certain elements
+differently between different modes or for different deployments.  A mode
+example might be the use of a custom overlay showing mode related DICOM header
+information such as radiation dose or patient age.
+
+The use of this service enables these to be defined in a typed fashion by
+providing an easy way to set default values for this, but to allow a
+non-default value to be specified by the configuration or mode.
+
+This service is a UI service in that part of the registration allows for registering
+UI components and types to deal with, but it does not directly provide an UI
+displayable elements unless customized to do so.
+
+## Registering Customizations
+There are several ways to register customizations.  The
+`APP_CONFIG.uiCustomizationService`
+field is used as a per-configuration entry.  This object can list single
+configurations by id, or it can list sets of customizations by referring to
+the `customizationModule` in an extension.
+
+As well, extensions can register default customizations by providing a 'default'
+name key within the extension.  These are simply customizations loaded when
+the extension is loaded, and applied before the white labeling.
```

Comment:
No white labeling now, also maybe give the default customization example right after this paragraph